### PR TITLE
Removed the duplication "confirmDelete" function on line 18

### PR DESCRIPTION
**Pull request by sriram7386**
The functionalities "confirmdelete" are being called by another function "openOkcancel" which results in duplication. The code has been refactored by changing the "openOkcancel " recurrences to "confirmDelete". The functionalities of "openOcancel" are not being used anywhere but only in this code segment. Duplication has been removed as code clone incur the maintainability costs and effort.

### CODE REVIEW

File: web/src/app/modules/notification/modules/modals/services/confirmation-modal.service.ts [hunk 1]
```diff
@@ -16,18 +16,18 @@ export class ConfirmationModal {
...
         const modalRef = this.modalService.open(TypedModalContent);
         modalRef.componentInstance.options = Dialogtype.okDialog(title, message);
         return modalRef.result;
     }
-
-    public openOkCancel(title: string, message: string): Promise<any> {
+  public openOkCancel(title: string, message: string): Promise<any> {
+        return this.change(title,message);
+    }
+    public change(title: string, message: string): Promise<any> {
```
Comment (munterkalmsteiner) [inline web/src/app/modules/notification/modules/modals/services/confirmation-modal.service.ts]: This method name needs to be more informative. Change to openOkCancel.
Also, there is no need for this method to be public.

File: web/src/app/modules/views/main/authentication/modules/login/components/login.component.css [hunk 2]
```diff
@@ -1,7 +1,12 @@
 .not-allowed {
     cursor: not-allowed;
 }
+.bgimg
```
Comment (munterkalmsteiner) [inline web/src/app/modules/views/main/authentication/modules/login/components/login.component.css]: Please create 1 pull request per feature (revert this change and open a new pull request).

**Comment by munterkalmsteiner** [comment]
Please see my previous comments. I cannot merge before they are not addressed.

File: web/src/app/modules/views/main/authentication/modules/login/components/login.component.css (hunk 2 above)
Comment (manpreethb) [reply web/src/app/modules/views/main/authentication/modules/login/components/login.component.css]: We have placed a new pull request titled "Feature-7 (Group-3)Background image for login page ".
Thank you.
